app/src/main/proto/vss.proto

```diff
@@ -293,7 +293,40 @@ message KeyValue {
   int64 version = 2;
 
   // Object value in bytes which is stored (in put) and fetched (in get).
-  // Clients must encrypt this blob client-side before sending it over the wire to server in order
-  // to preserve privacy and security.
+  // Clients must encrypt secret contents of this blob client-side before sending it over the wire
+  // to server in order to preserve privacy and security.
+  // Clients may use a `Storable` object on client-side and set it here.
```

Comment:
"client-side" seems redundant given the subject of the sentenced is "Clients". Instead, phrase it something like serializing the object before sending it to the server.